Scripts/script-CommonServer.yml

```diff
@@ -1839,6 +1839,19 @@ script: |-
 
       /**************************** HMAC_SHA256 end *******************************/
 
+    /**************************** REGEX FORMATTING *******************************/
+
+      var ipRegex = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
```

Comment:
You should use the regex from the ExtractIP script, and change the ExtractIP to use this regex...